friendships.go

```diff
@@ -0,0 +1,241 @@
+package twitter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+// NoRetweetIDs calls the Twitter /friendships/no_retweets/ids.json endpoint.
+func (c *Client) NoRetweetIDs(ctx context.Context, stringifyIDs bool) (*UserIDsResponse, error) {
+	values := url.Values{}
+	values.Set("stringify_ids", strconv.FormatBool(stringifyIDs))
+	resp, err := c.do(ctx, "GET", "https://api.twitter.com/1.1/friendships/no_retweets/ids.json", values)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	if err = checkResponse(resp); err != nil {
+		return nil, err
+	}
+	var userIDs []string
+	err = json.NewDecoder(resp.Body).Decode(&userIDs)
+	if err != nil {
+		return nil, err
+	}
+	return &UserIDsResponse{
+		UserIDs:   userIDs,
+		RateLimit: getRateLimit(resp.Header),
+	}, nil
+}
+
+// friendshipsDirection calls the Twitter /friendships/{direction}.json endpoint.
+func (c *Client) friendshipsDirection(ctx context.Context, direction string,
+	cursor string, stringifyIDs bool) (*UserIDsPageResponse, error) {
+	values := url.Values{}
+	if cursor != "" {
+		values.Set("cursor", cursor)
+	}
+	values.Set("stringify_ids", strconv.FormatBool(stringifyIDs))
```

Comment:
Again, if marshalling IDs to strings, the `"stringify_ids"` parameter should always be set to `"true"`.